sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,62 @@
+UPDATE `creature_template` SET `unit_flags`=33587264, `InhabitType`=4 WHERE `entry` IN (29310,31465); -- Jedoga
+UPDATE `creature` SET `MovementType`=0, `spawndist`=0 WHERE `guid`=131953;
+UPDATE `creature_addon` SET `auras`='56075 56327' WHERE `guid`=131953; -- Jedoga
+UPDATE `creature_template` SET `ScriptName`='npc_twilight_volunteer' WHERE `entry`=30385;
+UPDATE `creature_template` SET `ScriptName`='' WHERE `entry` IN(30114,30181);
+DELETE FROM `creature` WHERE `id` IN(30114,30181);
+DELETE FROM `creature_addon` WHERE `guid` IN (132063,132066,132067,132068,132069,132070,132071,132072,132065);
+DELETE FROM `linked_respawn` WHERE `guid` IN (131997,131998,131999,132000,132001,132002,132003,132004,132005,132006,132007,132008,132009,132010,132011);
```

Comment:
linked_respawn doesnt exist on head right?

Reply:
Still exists, will trigger a DB error if the values are kept there